benefits/core/context_processors.py

```diff
@@ -7,10 +7,17 @@
 from . import models, session
 
 
+def unique_values(original_list):
+    # dict.fromkeys gets the unique values and preserves order
+    return list(dict.fromkeys(original_list))
```

Comment:
Clever. I think we could use a quick test for this helper.

Reply:
Added in https://github.com/cal-itp/benefits/pull/1970/commits/f3fe500824bd906113038a2ef0c342c1da56f2d8